.github/CONTRIBUTING.rst

```diff
@@ -26,13 +26,16 @@ land a single large feature.
 
 Checklist:
 
-1. All pull requests *must* be made against the ``master`` branch.
-2. Include tests for any functionality you implement. Contributions that
+1. Always create a new branch to work on a new issue.
```

Comment:
I was also thinking to mention the naming convention of the branch that would created as in the name should be related to the changes you're doing. Also we could add a command to show how to create a new branch.

But felt like those information would be redundant, suggestions?

Reply:
Yes you can add the command `git checkout -b <branch-name>` . Helpful for new comers. We should actually add a getting started with git link somwhere.

Reply:
Added the command.

@ananyo2012 Let me know a good git getting started guide that you prefer, I will try to plug it in as note before the `Checklist`